Casks/y/yubico-yubikey-manager.rb

```diff
@@ -14,6 +14,8 @@
 
   no_autobump! because: :requires_manual_review
 
+  disable! date: "2025-02-19", because: :discontinued, replacement_cask: "yubico-authenticator"
```

Comment:
We should disable the cask with today's date, so it follows our usual deprecation schedule timeline.

```suggestion
  disable! date: "2025-07-27", because: :discontinued, replacement_cask: "yubico-authenticator"
```

Reply:
I see!